hivemq/assets/dashboards/hivemq.json

```diff
@@ -0,0 +1,682 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "HiveMQ",
+    "description": "The default dashboard for monitoring HiveMQ brokers including connections, subscriptions, and messages.",
```

Comment:
I think you also need a `created_by` section (https://github.com/DataDog/integrations-core/blob/c67d2041cb2cb9375ca3381b818468e29a8b546a/proxysql/assets/dashboards/overview.json#L7)

Reply:
I believe the field was removed in this PR as its not needed https://github.com/DataDog/integrations-core/pull/6787/s